src/db/sql_types.rs

```diff
@@ -9,11 +9,14 @@ pub enum ActionType {
     Delete,
 }
 
+/// What kind of data are we acting on? i.e. all
+/// data, our own data, some row?
 #[derive(Debug, DbEnum, Serialize, Deserialize)]
 #[DieselType = "Action_modifier"]
 pub enum ActionModifier {
     All,
     Own,
+    One,
```

Comment:
Did you change the corresponding DB type?

Reply:
Where should I do that?

Reply:
You'll need to make a migration. Unfortunately changing enums is a pain in the ass. Since migrations are run inside a transaction, you can't do the simple solution, so you have to go through the effort of creating a new enum, changing the names and swapping the column values. Very annoying. https://stackoverflow.com/questions/1771543/adding-a-new-value-to-an-existing-enum-type/7834949#7834949